inlong-manager/manager-command/src/main/java/org/apache/inlong/manager/command/CliCommand.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.command;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.apache.inlong.manager.client.api.ClientConfiguration;
+import org.apache.inlong.manager.client.api.auth.DefaultAuthentication;
+import org.apache.inlong.manager.client.api.impl.InlongClientImpl;
+import org.apache.inlong.manager.client.api.inner.InnerInlongManagerClient;
+
+import java.lang.reflect.Field;
+import java.util.List;
+
+abstract class CliCommand {
+
+    InnerInlongManagerClient connect() {
+        String serviceUrl = "127.0.0.1:8080";
```

Comment:
get the manager  api address from [conf/inlong.conf](https://github.com/apache/incubator-inlong/blob/master/conf/inlong.conf)

```
# manager service IP
manager_server_hostname=127.0.0.1
# manager port
manager_server_port=8083
```